spec/Controls/GeosearchSpec.js

```diff
@@ -53,4 +68,20 @@ describe('L.esri.Geosearch', function () {
     expect(map.attributionControl._container.innerHTML).to.contain('Geocoding by Esri');
   });
 
+  it('should call the set bounds', function (done) {
```

Comment:
in [`L.esri.Query`](https://github.com/Esri/esri-leaflet/blob/master/spec/Tasks/QuerySpec.js) tests we are just stubbing out a mock response from the server (rather than capturing it) to provide coverage for additional methods that post process responses.  sufficed to say, you needn't worry about expanding our test coverage here, what you have above is sufficient.